ext/drreg/drreg.dox

```diff
@@ -53,6 +53,11 @@ less available in future phases; the current logic skips over a slot if
 there's a usage found anywhere later in the bb added by any previous phase.
 So it requires additional spill slots as well.
 
+Note that DR provides APIs to directly access TLS slots: like
+\p dr_insert_write_raw_tls(), \p dr_insert_write_tls_field(). These are
```

Comment:
This is incorrect: dr_insert_write_tls_field uses the single client field which is unrelated to drreg (except via drmgr multiplexing it to provide drmgr_register_tls_field).

Reply:
dr_insert_write_raw_tls is perfectly safe to use, and many clients do use it, for slot ranges reserved by dr_raw_tls_calloc.  Only if someone used a slot range they did not actually reserve, which doesn't seem like we need to call it out here.

Reply:
The problem is if drreg is not given enough of its own slots via `#drreg_options_t.num_spill_slots` and so ends up using DR spill slots, and a client simultaneously directly uses DR's spill slots via dr_save_reg()/dr_restore_reg() (and the corresponding dr_read_saved_reg()/dr_write_saved_reg()).

Reply:
Thanks. Done.